waddrmgr/manager.go

```
@@ -1360,6 +1360,16 @@ func (m *Manager) Unlock(passphrase []byte) error {
 	return nil
 }
 
+// fetchUsed returns true if the provided address id was flagged used.
+func (m *Manager) fetchUsed(addressID []byte) bool {
+	var used bool
+	_ = m.namespace.View(func(tx walletdb.Tx) error {
```

Shouldn't ignore the error here.